### DISCUSSION TITLE
Model inconsistencies

### DISCUSSION THREAD

**Discussion by tillprochaska** [alephdata/followthemoney#948] [General]
Rather than creating a separate issue every time I encounter an inconsistency in the FtM data model, I’ll simply use this discussion to keep a list for future reference.

* https://github.com/alephdata/followthemoney/issues/924
* `Contract` has a `title` and a `name` (inherited from `Thing`).
* `Event` has a `location` property and `address`/`addressEntity` (inherited from `Thing`). See #952
* Names/labels of `Document` (label: "File") and `Pages` (label: "Document") are confusing

**Comment by yharahuts** [comment]
`LegalEntity` have it's phone number as `phone` property, and `UserAccount` have it's as a `number`.

I guess both properties should be same - `phoneNumber` maybe?

**Reply by pudo** [reply]
I'd vote making `UserAccount.phone`, since `UserAccount` is barely used anywhere.

**Comment by yharahuts** [comment]
`Post` have `organization` as string, while `Employment` have `employer` as entity link.

**Reply by pudo** [reply]
That's pretty consistent around all the link types (`director`, `owner`, `asset`) etc.

**Reply by yharahuts** [reply]
@pudo 
`Directorship` - `director` and `organization` is both entities (but a `secretary` isnt 🙄)
`Ownership` - `owner` and `asset` both entities
`Employment` - `employer` and `employee` both entities

So everything above can be used a link between Company and a Person.

`Post` - `holder` is entity, but `organization` is a string - so it can't be used as a link. Probably `organization` should be entity too?

**Reply by pudo** [reply]
No, that's the entire rationale for doing `Post` rather than `Employment`: to allow to depict people's CVs without ending up with dumb pseudo-entities: https://github.com/alephdata/followthemoney/issues/761

**Reply by yharahuts** [reply]
Oh I see now. 

Thank you for clarifying.
